src/NRediSearch/Client.cs

```diff
@@ -30,23 +31,21 @@ public enum IndexOptions
             /// <summary>
             /// The default indexing options - use term offsets and keep fields flags
             /// </summary>
-            Default = UseTermOffsets | KeepFieldFlags,
+            Default = UseTermOffsets | KeepFieldFlags | KeepTermFrequencies,
             /// <summary>
             /// If set, we keep an index of the top entries per term, allowing extremely fast single word queries
             /// regardless of index size, at the cost of more memory
             /// </summary>
-            UseScoreIndexes = 4,
-            /// <summary>
-            /// If set, we will disable the Stop-Words completely
-            /// </summary>
-            DisableStopWords = 8
```

Comment:
I don't know the back-story here; is this an API that got removed? or...?

The key point for me here is that this is a breaking change; that isn't necessarily a blocker, but there are usually better ways to manage a breaking change, for example adding `[Obsolete(..., true)]` for a while, and raising suitable exceptions - so that it isn't *as* surprising

Reply:
I readded the `DisableStopWords` enum. It was removed initially when I was going a bit mustang trying to match the JRediSearch api.